docs/main/prerequisites.md

```diff
@@ -34,37 +34,6 @@ The following operating systems are **officially** supported:
 
 Pi-hole only supports actively maintained versions of these systems.
 
-<!-- markdownlint-disable code-block-style -->
-!!! info
-    One of the first tasks the install script has is to determine your Operating System's compatibility with Pi-hole
-
-    It is possible that Pi-hole will install and run on variants of the above, but we cannot test them all. If you are using an operating system not on this list you may see the following message:
```

Comment:
I think we should keep part of the text to make clear Pi-hole may not work on every OS.

Something like this:

```md
Pi-hole may be able to install and run on variants of the above, but we cannot test all of them.
It's possible that that the installation may still fail due to an unsupported configuration or specific OS version.

Also, if you are using an operating system not on this list Pi-hole may not work.
```

Reply:
Re-added